packages/concatjs/web_test/test/karma/coverage.spec.js

```diff
@@ -0,0 +1,27 @@
+(function(factory) {
```

Comment:
I don't think you meant to check in these generated `.js` files?

Reply:
This might be me misunderstanding the test setup. I compared this with the other .js/.spec.js files in that directory.
Should I have added these tests differently?